Hosting: Update Dialog for phpMyAdmin Password Reset

Aurorum opened the pull request:
Fixes #95889 

## Proposed Changes

* Updates the "Reset password" confirmation dialog for the phpMyAdmin card. 

## Why are these changes being made?

The current dialog design is inconsistent with others. Most importantly, the buttons are the wrong way round, which can cause some confusion, but some of the spacing is also off. In addition, we have a `ConfirmDialog` component specifically for this purpose, so it makes sense to use it. 

## Testing Instructions

* Go to `/hosting-config/site.com` on an Atomic site 
* Navigate to Server Settings
* Click "Try resetting the password" under the phpMyAdmin card
* Confirm the new dialog works

| Before | After |
|--------|--------|
| <img width="616" alt="Screenshot 2024-10-31 at 23 06 44" src="https://github.com/user-attachments/assets/7ac06cbf-bcc7-450b-865f-b5295d1c4717"> | <img width="698" alt="Screenshot 2024-10-31 at 23 01 43" src="https://github.com/user-attachments/assets/100e37ec-4829-40ff-ac52-47f06595c0f0"> | 

cc @nightnei, @wongasy, @fushar

Aurorum committed:
Hosting: Update phpMyAdmin Card Dialog

Aurorum committed:
Hosting: Update phpMyAdmin Card Dialog

Aurorum committed:
Hosting: Update phpMyAdmin Card Dialog

kangzj committed:
Stats: add start date to query (#95919)

* add start_date to query

* always pass summarize=1 for summary page link

* flip the feature flag to false for development env

* revert the change to add summarize

lsl committed:
Remove center justification on panel content (#95852)

wongasy commented:
Thanks @Aurorum for putting in a fix! We'll need specs from the design team to ensure the modal component renders with a consistent look in all use cases. I'll get back to you once we get the specs.

wongasy commented:
My apologies, it looks like I spoke too early! I noticed that you're leverage an existing `<ConfirmDialog>`, which should give us a consistent look. I'll double check with the team.

zaerl committed:
Add missing page tracks (#95878)

kangzj committed:
hide email stats when new date filtering enabled (#95975)

kangzj committed:
Stats: add start date for devices and utm stats (#95924)

* always pass summarize=1 for summary page link

* devices

* suppor start date for utm

* suppor start date for utm

* revert summarize param

* default start date to empty string

* Apply suggestions from code review

* Update client/my-sites/stats/hooks/utils.ts

kangzj committed:
we should include the start date, so the days should be taken away 1 (#95974)

arthur791004 committed:
SMP: Make the item preview content with sidebar responsive (#95874)

* SMP: Make the sidebar of the item preview content sticky

* SMP: Enforce the tabs view

* SMP: Use dropdown for the 2nd level tab on mobile

* Fix the box-shadhow of the tabs on mobile

* Change breakpoint and fix the width of site-settings

* Fix tests

zaerl committed:
Add missing monitoring and logs tracking events (#95941)

zaerl committed:
Add missing overview tracking events (#95883)

* Add missing quick action tracks

* Add missing onClick callback

* Add missing tracks

* Fix: wrong import

* Fix type error

Uses of `<Action>` that don't include a `commandName` shouldn't send a
tracks event, because we won't know how to tell them apart.

---------

Co-authored-by: Philip Jackson <p-jackson@live.com>

fushar committed:
Untangling: fix focus style on panel sidebar (#95940)

lsl committed:
Site settings page headers (#95936)

* Site settings page headers

* JSX->TSX + header title logic improvement

* Fix types

jeyip committed:
Bluehost Plans: Remove definitions from Calypso (#92773)

lsl committed:
Ensure site prop is not null (#95981)

fushar committed:
Untangling: redirect to /hosting-features if Advanced Tools is not supported (#95978)

fushar committed:
Untangling: show unsupported notice for Settings -> Administration for staging sites (#95937)

* Untangling: show unsupported notice for Settings -> Administration for staging

* Merge fix: add header back

---------

Co-authored-by: Louis Laugesen <louis.laugesen@gmail.com>

lsl committed:
Add agency settings toggle (#95934)

* Agency toggle

* Empty state

* Title

* Typescript after rebase

---------

Co-authored-by: Ashar Fuadi <ashar.fuadi@automattic.com>

arthur791004 committed:
wpcom-block-editor: Support getting the canvas mode from the query string after GB 19.6 (#95979)

* wpcom-block-editor: Support getting the canvas mode from the query string after GB 19.6

* wpcom-block-editor: Simplify the logic of useCanvasMode

arcangelini committed:
Help Center: fix aggressive styles (#95953)

eoigal committed:
Reader: update reader sidebar icons (#95909)

* Reader: update search icon

* update discover icon

* a lot of sidebar icon changes

* update more stuff

wongasy commented:
@Aurorum I checked with the design team. We are actually using the correct [`<Dialog>` component](https://github.com/Automattic/wp-calypso/tree/trunk/packages/components/src/dialog) in production. 

To fix the issue for the phpMyAdmin password reset modal, we need to:
* Update the styles in the `<Dialog>` component
* Switch the order of the buttons so the primary action sits on the right, as you had pointed out
* Add the `showCloseIcon` property to ensure the "x" button shows up

Commit:
Help Center: Fix avatar size (#95986)

* Fix avatar size

* Remove unnecessary classed

DustyReagan committed:
Move continue button to modal header (#95967)

DustyReagan committed:
Reader Onboarding - Ship to users by removing the `reader/onboarding` flag (#95914)

* Remove the reader/onboarding flag

* Remove Following Intro banner

eoigal committed:
Reader: update post actions positioning (#95987)

cpapazoglou committed:
Cleans up an October 2024 Facebook placement. (#95990)

Co-authored-by: cpap <papazoglou.charalampos@gmail.com>

eoigal committed:
Sites: fix positioning of p2 menu item (#95989)

eoigal committed:
Reader: update ellipsis menu order (#95993)

sirbrillig committed:
Checkout: Prevent fatal in getPlanFeatures for unlisted product (#95827)

leonardost committed:
Domains: Hide cancel and transfer options for 100-year domains (#95907)

* Hide transfer and cancel buttons in the domain management page for 100-year domains

* Hide cancel option in purchase management page for 100-year domains

* Add `HundredYearDomainNotTransferrableNotice` component and refactor notices

* Redirect from purchase cancelation page if 100-year domain

If you go to the purchase cancelation page for a 100-year domain, you
are now redirected to the purchase management page. We don't want to
let users turn auto-renew off or cancel these domains.

fditrapani committed:
Update copy (#95838)

andres-blanco committed:
check for translation before using it (#96000)

Co-authored-by: Andrés Blanco <email@gmail.com>

Aurorum committed:
Address review

Aurorum committed:
Address review

Aurorum committed:
Address review

Aurorum commented:
Ah, thanks for checking! It looks like the issue with spacing was caused by some bleeding styles which I've tidied up here. 

<img width="623" alt="Screenshot 2024-11-02 at 11 14 34" src="https://github.com/user-attachments/assets/68762781-77fa-49f2-be44-4f4e5b0dccb4">

I've also reversed the button order and added a close icon.

Commit:
Help Center Remove history icon from the header on chat route (#95992)

d-alleyne committed:
Fix some theme filters appearing untranslated (#95995)

annacmc committed:
Stats: Chart refresh header component update (#95942)

* create new chart instance behind feature flag

* adds a className prop to the chart

* Add ChartHeader component for better composition

* update show/hide legend prop for legacy vs new chart

* Adds dedicated header component with title and controls props

* Reorganise header layout

* Adds activeTab to the chart title

* remove control section for a future PR

* add a showChartHeader prop

* small fix to header title display

* add in accidentally removed comment

jkguidaven committed:
A4A: Implement the 'Hosting options' section on the new Migrations page. (#95939)

* Implement Hosting options section on the new Migrations page.

* Add footnote.

* Make button responsive.

* Replace WPCOM banner image.

yashwin committed:
A4A > Overview: Fix the growth accelerator overflow issue (#95790)

jkguidaven committed:
A4A: Implement the testimonials section on the new Migrations page. (#95859)

* Implement the Testimonials section on the new Migrations page.

* Add the new background.

yashwin committed:
A4A > Marketplace: Open Pressable support when the Contact us button is clicked (#95797)

* Open Pressable support for when Contact us button is clicked

* Update variable name

jkguidaven committed:
A4A: Implement the 'Migrations process' section on the new Migrations page. (#95809)

* Implement the 'Migrations process' section on the new Migrations page.

* Add the new background.

jkguidaven committed:
A4A: Implement the 'Client relationship' section on the new Migrations page. (#95807)

* Implement the 'Client relationship' section on the new Migrations page.

* Make sure styling matches design.

jkguidaven committed:
A4A: Implement the CTA section on the new Migrations page. (#95804)

* Implement the CTA section on the new Migrations page.

* Make the CTA button responsive.

Commit:
Help Center: Add long message validation error (#95985)

* Add long message validation error

* Add long message validation error

* Remove important

* Rename variable

* Applying suggested refactor

jkguidaven committed:
A4A: Implement the banner section for the new migrations page. (#95794)

* Move SimpleList component to global shared components.

* Implement main banner.

* Add new background.

* Address PR comments.

jkguidaven committed:
A4A: Implement the 'Hosting features' section on the new Migrations page. (#95802)

bogiii committed:
Photo Picker: Update migration connection notification (#96005)

jkguidaven committed:
A4A: Implement the FAQs section on the new Migrations page. (#96017)

* Implement the FAQs section in the new Migrations page.

* Fix few grammars.

* Add FAQ for buyout credit calculation.

* Address PR comments.

* Address more comments.

hambai committed:
Send the vendor string to the domain availability check to support 100 year domain prices (#95906)

We need to get the correct price for 100 year domains in the special 100 year domain flow. In order to do that we need to send the vendor string to the availability endpoint.

wongasy commented on client/blocks/site-address-changer/style.scss:
```diff
@@ -5,14 +5,13 @@
-.dialog__action-buttons {
-	border: 0;
-	padding: 35px 48px 48px;
+	.dialog__action-buttons {
+		border: 0;
+		padding: 35px 48px 48px;
```
@Aurorum Looks like we'll break the style of `<SiteAddressChanger>` with this change. Here's how you'd test the site address changer: https://wordpress.com/support/changing-site-address/#change-your-word-press-com-site-address

|before|after|
|-------|-----------------|
|<img width="848" alt="Screen Shot 2024-11-04 at 8 51 24 AM" src="https://github.com/user-attachments/assets/f5abb5a7-b06e-4378-9fb8-4b2cef0a39e3">|<img width="858" alt="Screen Shot 2024-11-04 at 8 51 19 AM" src="https://github.com/user-attachments/assets/c168faa1-a8a0-43c6-a2be-b0a2f7936468">|

To we can customize the button style for this dialog box, we may consider:

* Wrapping the `<Dialog>` component for the `<SiteAddressChanger> [here](https://github.com/Automattic/wp-calypso/blob/d011f3b3dd669d52a300b3923faced62280f0154/client/blocks/site-address-changer/index.jsx#L554) with a div, assign a custom class name to it.

* Updating `<Dialog>` to apply the custom class name to the [<ButtonBar>](https://github.com/Automattic/wp-calypso/blob/d011f3b3dd669d52a300b3923faced62280f0154/packages/components/src/dialog/index.tsx#L100).

Aurorum replied:
Thanks @wongasy - very good catch! I'd appreciate your guidance here on whether we even need these additional styles though. If the aim is consistency of Dialogs across Calypso, I feel like `<SiteAddressChanger/>` might work best without the forced extra padding?

<img width="774" alt="Screenshot 2024-11-05 at 16 58 25" src="https://github.com/user-attachments/assets/64b72b0b-ed82-4884-a409-c86de512846b">

I've committed it for now, but let me know and I can change it as necessary.

wongasy replied:
Ah, is the screenshot you shared the outcome of removing `className="site-address-changer"`? If so, I think it's the right call to to stick with the base style from `<Dialog>`!

Aurorum replied:
Ah, perfect - yep, the screenshot is with the class name removed and just the default Dialog styles. :)

heavyweight committed:
Help Center: Fix autoscroll issues (#95712)

* restructure lastMessage management

* restructure leftovers

* simplify autoscroll

* manage loaded state

* Fix type error

* Fix merge issue

---------

Co-authored-by: Tony Arcangelini <tony@arcangelini.com>
Co-authored-by: Renan <renansscarvalho@gmail.com>

IanRamosC committed:
Jetpack Cloud: allow redirect on free button from My Jetpack (#96004)

Commit:
Help Center: Fix avatar image on chat history (#96020)

* Fix avatar image on chat history

* Use user gravatar on chat history

* Increase gravatar image size

agrullon95 committed:
Help Center: Add logic for archived conversations (#95839)

* Add logic for archived conversations

* Refactor conversations for reusable

* Refactor focus styles in chat history and section navigation

* Remove empty line

* Remove unnecessary return statements

* Remove tab on empty line

* Extract lastMessage function and sort recieved messages by unread + last message

* fix utils function errors

* Fix lint issues

heavyweight committed:
Navigate to conversation when there's one unread conversation (#96019)

sixhours committed:
Post-migration purchase experience: Add "View sites dashboard" button to migration message screen (#95762)

* Add sites dashboard button to migration message screen

* Fix capitalization

* Remove cards, update copy

sixhours committed:
Post-migration purchase experience: Add "View Sites dashboard" button to migration started page (#95761)

* Add view sites dashboard button

* Further updates to better match design.

* Update capitalization

* Capitalization

arcangelini committed:
Odie: add support interactions API (#95996)

epeicher committed:
Performance Profiler: Update the weekly report text (#96023)

andres-blanco committed:
Post-purchase experience: remove pending-migration sticker (#95994)

* Post-purchase experience: remove pending-migration sticker

* prevent setting any sticker

* remove tests

* remove more tests

* remove tests

---------

Co-authored-by: Andrés Blanco <email@gmail.com>

sirbrillig committed:
Purchases and Checkout: Rename createStripeSetupIntent to confirmStripeSetupIntentAndAttachCard for clarity (#96029)

* Rename createStripeSetupIntent as confirmStripeSetupIntent

* Use confirmStripeSetupIntent in assignNewCardProcessor

* Use confirmStripeSetupIntent in jetpack-cloud

* Use confirmStripeSetupIntent in a4a

* Deprecate StripeSetupIntentIdProvider

epeicher committed:
Perofrmance Profiler: Open Learn More from Migration banner in a new window (#96028)

dereksmart committed:
Store site goals in Big Sky launch step (#95969)

dereksmart committed:
Show Big Sky only on desktop and tablet (#95965)

holdercp committed:
Help Center: use URL params for determining editor canvas state (#95964)

* Help Center: Use router params for canvas mode

* Add wordpress router dep

* Dedupe wordpress router

* Only return canvas mode value in the site editor

* Match wordpress router version of wpcom-block-editor

* Reset yarn.lock from trunk

allilevine committed:
Reader: Stub out DataViews for Recent Feed Overhaul (#95727)

* Stub out DataViews for Recent Feed Overhaul.

StevenPartridge committed:
Moves contact validity message up a component, introducing it to both the active and completed state (#95972)

Co-authored-by: Steven Partridge <steven.partridge@automattic.com>

paulopmt1 committed:
Increase seria scale from 50 to 70px to accommodate bigger numbers (#96033)

DustyReagan committed:
Strip HTML from site description (#96030)

aneeshd16 committed:
Plans: ship the simplified grid to all users (#96001)

kangzj committed:
Stats: Change width of most cards to 50% for easier layout (#95976)

* why not half & half?

* remove more unused code

* use a variable for haf width modules

* className should be string

mreishus committed:
Revert "Help Center: use URL params for determining editor canvas state (#95964)" (#96042)

This reverts commit b4cbdd855072fdffaca8a14cf18996ac665c5cb7.

kangzj committed:
Stats: Fix module setting popup width (#96039)

fushar committed:
Untangling: Add Advanced Tools -> Monitoring (#96015)

fushar committed:
Untangling: Add Advanced Tools -> Logs (#96016)

fushar committed:
Untangling: refactor feature gating so that sidebar and content use same logic (#95983)

holdercp committed:
Help Center: use URL params for canvas state (#96046)

* Use url params for canvas state

* Provide default for useLocation

fushar committed:
Untangling: Add Advanced Tools -> Staging Site (#96021)

heavyweight committed:
Help Center: style archived no results (#96026)

* HelpCenter: style archived no results

* Add translation domain. Change url

---------

Co-authored-by: escapemanuele <escapemanuele@gmail.com>

taipeicoder committed:
Media: Update video/audio uploads upsell nudge (#96024)

Commit:
Help Center: Fix send message button disabled state (#96052)

yashwin committed:
A4A > Overview: Update the Pressable card text (#95783)

yashwin committed:
A4A > Add Site: Update text & order of the "Add site" menu (#95784)

* Update text & order of Add site menu

* Use preventWidows to prevent runts

rcrdortiz committed:
Fixed the event props so that the event does not end up rejected (#96054)

yashwin committed:
A4A > Checkout: Update referral checkout summary notice (#95787)

escapemanuele committed:
Help Center: Update chat paragraphs text spacing (#96062)

* Remove the huge whitespace

* Remove padding from p

jkguidaven committed:
A4A: Enable the new Migrations page on the 'Staging' and 'Production' environment. (#96047)

simison committed:
Newsletter settings: add floating subscribe button (#96022)

bogiii committed:
Photos Picker: Introduce Google Photos Picker UI changes (#95880)

* Update endpoints to match a new setup

* Create GooglePhotosPickerButton component

* Update HOC providing more props

* Fix wrong typing

* Integrate PhotosPicker into media libriary component

* Adjust media-items fetching different use-cases

* Hide header until we get media items

* Rename from photo to photos to be more consistent

* Add 'Change selection' action

* Adjust google photos picker screen and add translation

* Create google proxy media request functions

* Create and integrate the GoogleProxiedImage component

* Delete session on item copy event

* Hide Open PhotosPicker button if the feature flag is not there

* Turn off the feature flag until we prepare the whole flow

* Improve useGoogleProxy condition

* Improve condition

* Remove delete picker session on file copy event

* Delete picker session when user click on change selection CTA

* Adjust Google Photos empty state

* Improve the condition, making sure that the feature flag check exists

bogiii committed:
Photos Picker: Google Photos: Hide filtering if feature flag is ON (#96009)

Feature flag: google-photos-picker

escapemanuele committed:
Help Center: Fix wrap for long words (#96063)

bogiii committed:
Photos Picker: Don't provide filter options when the google-photos-picker is turned on (#96025)

paulopmt1 committed:
Feature flag for Jetpack cloud added (#96027)

* Feature flag for Jetpack cloud added

* Compatibility adjustments

* Add siteObjectsToSiteIds computing to the old interface only

yuliyan committed:
e2e: Fix block highlight step in i18n__editor spec (#96067)

* e2e: Fix block highlight step in i18n__editor spec

* Remove click action as the block is already auto selected after insertion

DustyReagan committed:
Recent Feed Overhaul - Use the FullPostView component (#96043)

* Use the FullPostView component

* Simplify code

fditrapani committed:
Update 100-Year domains (#95997)

* Update 100-Year domains

* Hide transfer link

Commit:
Help Center: Load attachments from zendesk (#96068)

* Allow images to be loaded

* Handle links and text

* Handle more complex URLs

* Fix the import

* Fix type check

* Add a type back

* Add generic message for not support message types

* Handle other file types

---------

Co-authored-by: escapemanuele <escapemanuele@gmail.com>

escapemanuele committed:
Help Center: Remove disclaimer if not needed (#96078)

agrullon95 committed:
Help Center: prevent new conversation initiation if already started and disable button accordingly (#96074)

Aurorum committed:
Remove additional Site Address Changer styles

Aurorum committed:
Remove additional Site Address Changer styles

agrullon95 committed:
Help Center: Update button text from 'Start conversation' to 'Get support' (#96081)

gcsecsey committed:
use a hardcoded list of high impact audits (#96073)

eoigal committed:
Reader: add recent stream (#96066)

* Reader: add recent stream

* fix post keys

* remove paging

kthai15 committed:
delete boilerplate about page if launching Big Sky (#96041)

wongasy reviewed (approved):
@Aurorum The changes work as expected!

|phpMyAdmin Password Reset|Site Address Change|
|-----------------------------|----------------------|
|<img width="612" alt="Screen Shot 2024-11-05 at 3 41 43 PM" src="https://github.com/user-attachments/assets/54a586e9-b937-49a8-8902-0177aec778c9">|<img width="764" alt="Screen Shot 2024-11-05 at 3 39 39 PM" src="https://github.com/user-attachments/assets/92339dc4-a4c8-4178-b0a1-0442ffb76640">|

If you could add the `showCloseIcon` property to the dialog box of `<SiteAddressChanger>`, we're good to go!

Aurorum committed:
Add showCloseIcon to Site Address Changer

Aurorum committed:
Add showCloseIcon to Site Address Changer

Aurorum commented:
Thanks @wongasy - all done!

kangzj committed:
alway use day for summarize (#96089)

lsl committed:
Move domain search suggestion onClick to Select Button (#96013)

* Move domain search onClick to Button

* Update e2e's domain-search-component.ts::selectDomain

* Restore waitFor calls

* Clean up awaits

phcp committed:
Domains: Fix the spacing on Domains page to match with Sites page (#96086)

* Add domains-table-search className

* Fix domains-table position

* Fix toolbar and table header spacing

* Fix navigation header spacing

* Fix search component spacing

* Mobile touchups

* Figured out what the important was doing

---------

Co-authored-by: Louis Laugesen <louis.laugesen@gmail.com>

fushar committed:
Untangling: Add Advanced Tools -> Deployments (#96059)

yashwin committed:
A4A > Signup: Adding smoke tests (#94747)

* Add smoke test to A4A signup page

* address PR comments

* use label

* verify form values

fushar committed:
Untangling: Add Advanced Tools -> SFTP/SSH (#96064)

arthur791004 committed:
Assets: Fix the order of the chunk files (#95748)

* Assets: Fix the order of the chunk files

* Get rid of fallback

* Fix tests

* Update copy

arthur791004 committed:
Revert "Assets: Fix the order of the chunk files (#95748)" (#96099)

This reverts commit 0c5b372d9177d6d673f19524e53cabd7066c769c.

yuliyan committed:
Docs: Add subsection for `i18n-calypso` best practices (#96018)

yashwin committed:
A4A > Referrals: Enable Enterprise hosting for refer mode (#96045)

* Enable Enterprise for refer mode

* minor ui fix

rcrdortiz committed:
Global Styles on Personal: Update upsells and modals (#96053)

* Updated the upsell modals and theme details upsell messages

* Updated modals on privacy settings and launchpad

* Added text for themes details pages

* Changed the checkout page of the LiTS theme details page activate design

matticbot committed:
fix(deps): update dependency http-proxy-middleware to v3.0.3 [security] (#95595)

matticbot committed:
fix(deps): update dependency cookie to ^0.7.0 [security] (#95193)

Co-authored-by: Renovate Bot <bot@renovateapp.com>

agrullon95 committed:
Help Center: Adjust spinner position and margin in send message input (#96088)

* Adjust spinner position and margin in send message input

* Fix margin syntax in send message input spinner

* Fix margin syntax in help center chat input spinner

---------

Co-authored-by: Renan <renansscarvalho@gmail.com>

fushar committed:
Untangling: Add Advanced Tools -> Database (#96098)

tiagonoronha committed:
Integrate Pressable Intercom with A4A migration support form (#96104)

matticbot committed:
fix(deps): update dependency uplot to v1.6.31 [security] (#95073)

Co-authored-by: Renovate Bot <bot@renovateapp.com>

paulopmt1 committed:
Limit parallel DotOrg API requests (#96087)

* Requesting one Dotorg plugin data each 200ms

* Fix update button CTA showing for non updates state

* Flag added for bulk-plugin-management

* Batching 5 plugin requests in parallel

* Increased time to 500ms instead of 200

* Improved how we check for bulk management feature during dotorg loading

* Removed unnecessary check

vykes-mac committed:
fix possible re-renders with dataviews (#96094)

wongasy commented:
@Aurorum I was planning to deploy this today, but it looks like we have a merge conflict. Could you resolve it and I'll deploy the change today!

Commit:
Help Center: Fix history message attachment (#96110)

lezama committed:
Summary: Show clearer number of users imported/failed after import (#96014)

Co-authored-by: Mikael Korpela <mikael@ihminen.org>

gabrielcaires committed:
Post Purchase: Add feature flag pending status (#96102)

* Add feature flag

* Enable on wpcalypso

sirbrillig committed:
Payment Management and Checkout: Use dedicated endpoint to create Setup Intent (#96082)

* Change which endpoint is used to create Setup Intent

* Remove unused isCheckout argument

allilevine committed:
Reader Recent Feed Overhaul: Populate posts in middle column (#96071)

* Break out new RecentPostField component.

* Add featured image and formatting.

* Change to table view with combined fields and add seen field.

* Update for new endpoint.

* Use DataViews filterSortAndPaginate.

* Uncombine fields and clean up styles.

* Align post info and featured image.

* Prevent DataViews actions overflow.

* Use rem utility instead of ignoring style rule.

* Use iconSize prop instead of adding a new one.

holdercp committed:
Reader: move followed sites to left sidebar under Recent (#96040)

kthai15 committed:
remove personal plan from big sky eligibility (#96119)

paulopmt1 committed:
Performance improvements for Bulk Plugin Management (#96121)

* Initial improvements to prevent unnecessary re-renders

* Improve feature performance:
- Showing Global plugins updates only when having less than 100 plugins
- Using pluginsOnSites only for the old version of the feature
- Improved selector cache

* Restored changes from Update Plugins CTA

* Removed Update all plugins CTA

lsl committed:
Expand site_name acceptable types to include boolean accounting for purged sites (#96096)

* Expand site_name acceptable types to include boolean accounting for purged sites

* Another one

fushar committed:
Untangling: Add Settings -> Web server subtab (#96108)

yashwin committed:
A4A > Sites: Fix the site overview flyout panel URL reset (#96095)

fushar committed:
Untangling: Add Settings -> Caches subtab (#96130)

madhusudhand committed:
remove simple text link from marketing tools (#96101)

fushar committed:
Untangling: Hide Server Settings tab under feature flag (#96134)

escapemanuele committed:
Help Center: Update copy (#96076)

* Update copy

* Update help center copies with flag

* Change the right component

* Fix Odie initial message

* Help is on the way

* Change header text

* Minimized History

* Less Odie in HC

---------

Co-authored-by: Anthony Grullon <agrullon95@gmail.com>

annacmc committed:
Stats: Chart refresh- CSS styling (#95980)

* create new chart instance behind feature flag

* adds a className prop to the chart

* Add ChartHeader component for better composition

* update show/hide legend prop for legacy vs new chart

* Adds dedicated header component with title and controls props

* Reorganise header layout

* Adds activeTab to the chart title

* remove control section for a future PR

* add a showChartHeader prop

* small fix to header title display

* add in accidentally removed comment

* add date filtering enabled CSS classname

* add chart styling

* remove !importants, increase title font size

* adjust font weight

* update axis marger border color

* fix the code styling

heavyweight committed:
Help Center: add image upload for chats (#96105)

* HelpCenter: add upload (wip)

* Minor fixes and refactoring

* Changed icon

* Use correct constants

* Improve isAttachingFile state

* Store global clientId as fallback

* Fix attach button style

* Move up the flag check

* Improved function

---------

Co-authored-by: escapemanuele <escapemanuele@gmail.com>
Co-authored-by: Anthony Grullon <agrullon95@gmail.com>
Co-authored-by: Renan <renansscarvalho@gmail.com>

simison committed:
Importer: use "import" term for CTA rather than "Start import" (#96111)

epeicher committed:
Performance Profiler: Consolidate all time metrics to seconds (#96140)

fushar committed:
Untangling: Add Marketing -> Connections (#96135)

yuliyan committed:
localizeUrl: Update wp-login rule to support Romanian and Vietnamese (#95943)

Commit:
Help Center: Add temp flag for showing simplified chat history (#96139)

* Add temporary flag for displaying simplified chat history

* Improve border

phcp committed:
Fix duplicated email verified notice (#96125)

wongasy commented:
@Aurorum Hello there! Following up again to see if you could resolve the merge so I can deploy this fix, thanks!!

Aurorum committed:
Merge branch 'update/phpmyadmin-dialog' of https://github.com/Aurorum/a-calypso-once-more into update/phpmyadmin-dialog

Aurorum committed:
Merge branch 'update/phpmyadmin-dialog' of https://github.com/Aurorum/a-calypso-once-more into update/phpmyadmin-dialog

Aurorum commented:
oh dear something has gone quite wrong there...let me try this one again! And apologies for the pings to the codeowners...

davemart-in commented:
No worries! It happens!
